common/src/main/java/dev/architectury/registry/level/entity/trade/SimpleTrade.java

```diff
@@ -37,14 +37,9 @@
  * or
  * {@link TradeRegistry#registerTradeForWanderingTrader(boolean, VillagerTrades.ItemListing...)}.
  */
-public class SimpleTrade implements VillagerTrades.ItemListing {
-    private final ItemStack primaryPrice;
-    private final ItemStack secondaryPrice;
-    private final ItemStack sale;
-    private final int maxTrades;
-    private final int experiencePoints;
-    private final float priceMultiplier;
-    
+public record SimpleTrade(ItemStack primaryPrice, ItemStack secondaryPrice,
```

Comment:
Good catch! Might be worth Java 16-ifying the entire project a bit later down the line